server/single/.env.example

```
@@ -1,4 +1,5 @@
 BASE_URL=https://vcknots.example.com
 PORT=8080
+
 PRIVATE_KEY_PATH=../samples/certificate-openid-test/private_key_openid.pem
-CERTIFICATE_PATH=../samples/certificate-openid-test/certificate_openid.pem
\ No newline at end of file
+CERTIFICATE_PATH=../samples/certificate-openid-test/certificate_openid.pem
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CERTIFICATE_PATH key should go before the PRIVATE_KEY_PATH key

(UnorderedKey)